src/server/ua_subscription_datachange.c

```diff
@@ -10,6 +10,7 @@
 #include "ua_subscription.h"
 #include "ua_server_internal.h"
 #include "ua_types_encoding_binary.h"
+#include <math.h>
```

Comment:
Do we have math library on every architecture, or is it dependent on the architecture? Not sure about that. Anyways global includes should be before local includes. (I.e. this should be the first include)